react/src/lib/components/SubsurfaceViewer/components/Map.tsx

```diff
@@ -362,8 +369,9 @@ const Map: React.FC<MapProps> = ({
         viewPortMargins,
         boundsInitial,
         boundingBoxCenter(reportedBoundingBoxAcc),
-        views?.viewports?.[0].target,
-        views?.viewports?.[0].zoom,
+        views?.viewports?.[0],
+        views?.layout,
+        marginPixels ?? 0,
```

Comment:
You can set the default value in the `Map` or `SubsurfaceViewer` initialization.